index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3167ce6dab.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="header container">
+            <h1 class="header-heading">My logo</h1>
+            <nav>
+                <i class="header-icon fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    <section class="section-info">
+        <div class="info-display container">
+            <h2 class="info-heading">I´m Maury <br> Glad to see you!</h2>
```

Comment:
- [ ] The main title is almost there! However, it would be better to use an `h1` tag as this is the most important title of the document.

- [ ]  In order to make your design look more similar to the Figma template let's use the right font (Poppins).